### ISSUE TITLE
WCAG 2.1 AAA contrast ratio

### ISSUE THREAD

**Issue by JuliannaR** [veteransaffairscanada/vac-benefits-directory#1810]
In keeping with striving for WCAG 2.1 AAA this is something we can look at.

**1.4.6 Contrast (Enhanced)**
The color of the text and the color of the background are not in sufficient contrast to each other.

The contrast ratio should be at least 7:1 for normal text and 4.5:1 for large text.

Note: Contrasts are not relevant where content is hidden. But make sure that for hidden content, which becomes visible when receiving focus, the contrast ratio is sufficient.

Insufficient contrast between text and its background can give problems for users with visual impairments and color blindness, but it can also affect a lot of other users, leading to important text potentially being overlooked.

**How do I meet this criteria?**
G17 Ensuring that a contrast ratio of at least 7:1 exists between text (and images of text) and background behind the text
G18 Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text

<img width="1413" alt="screen shot 2019-02-05 at 8 09 28 pm" src="https://user-images.githubusercontent.com/35537308/52314739-f9348700-2981-11e9-82da-655b0f7d01ee.png">

**Comment by brdunfield** [comment]
Checked against new design and found these violations:
<img width="1358" alt="screen shot 2019-03-04 at 2 53 42 pm" src="https://user-images.githubusercontent.com/5032149/53760050-c651b600-3e8f-11e9-945e-1935106bd12b.png">
<img width="1358" alt="screen shot 2019-03-04 at 2 53 59 pm" src="https://user-images.githubusercontent.com/5032149/53760051-c651b600-3e8f-11e9-8810-ddad37920210.png">

Flagged to Design
Phil's response: for both those titles you can use hex 2F3437

He will be updating Zepplin mockups
